docs/submit-transactions.md

```diff
@@ -0,0 +1,156 @@
+---
+id: submit-transactions
+title: Submit transactions
+sidebar_label: Submit transactions
+---
+
+⚠ Before proceeding, make sure you have [installed the required binaries](getting-started#installation)
+and have followed the getting started steps regarding [submitting a transaction](getting-started#submitting-a-transaction).
+
+
+## Available transactions
+
+Discover all the available transactions that you can submit via the CLI.
+Run the provided help commands to learn more about the arguments
+and behaviour of each one of them.
+
+ⓘ All the transactions provide the same [Command-Line Options](#command-line-options).
+
+- Transfer funds from an account to another
+
+    ``` bash
+    radicle-registry-cli account transfer --help
+    ```
+
+- Transfer funds from an org's account to another account
+
+    ``` bash
+    radicle-registry-cli org transfer --help
+    ```
+
+- Register entities
+
+    You can register users, orgs, and projects in the Radicle Registry.
+
+    - Register a user
+        ``` bash
+        radicle-registry-cli user register --help
+        ```
+
+    - Register an org
+        ``` bash
+        radicle-registry-cli org register --help
+        ```
+
+    - Register a project
+        ``` bash
+        radicle-registry-cli project register --help
+        ```
+
+- Unregister entities
+
+    You can unregister orgs and users from the Radicle Registry.
+
+    - Unregister a user
+        ``` bash
+        radicle-registry-cli user unregister --help
+        ```
+
+    - Unregister an org
+        ``` bash
+        radicle-registry-cli org unregister --help
+        ```
+
+### Examples
+
+1. Register a user
+
+    This example registers a user named "neo" in the registry using a
+    locally generated account (that happens to also be) named "neo"
+    to sign the transactions, specifying `100` as the fee.
+
+    ``` bash
+    radicle-registry-cli user register neo --author neo --fee 100
+    ```
+
+2. Transfer funds from an account to another
+
+    This example transfers `500` tokens to the account with the address
+    `5HWP48i9TuP2VrRZrNeb6QzYpdUSE9BvyaptEqqHrLfH8ZPd`, specifying `20` as fee. A total of
+    `520` will be withdrawn from the author's account, neo.
+
+    ``` bash
+    radicle-registry-cli account transfer 500 5HWP48i9TuP2VrRZrNeb6QzYpdUSE9BvyaptEqqHrLfH8ZPd --author neo --fee 20
```

Comment:
The user needs to scroll to see all the code. We should break it into a couple of lines.